data/data-pipeline/data_pipeline/content/config/excel.yml

```diff
@@ -165,6 +165,12 @@ sheets:
       - score_name: Median value ($) of owner-occupied housing units
         label: Median value ($) of owner-occupied housing units
         format: float
+      - score_name: Share of homes with no kitchen or indoor plumbing (percentile)
```

Comment:
This is the opposite order of the other file -- is that intentional?

Reply:
Good catch! It isn't. I also had to make a bunch of field order changes that are gonna end up causing conflicts, so will fix, but in #1921